src/reducers/initialState.js

```diff
@@ -2,7 +2,8 @@ const initialState = {
   stuff: [],
   edit: {
     isEditing: false
-  }
+  },
+  pois: { asdf: 1 }
```

Comment:
this should follow the store structure defined in [slack](https://hack4impact.slack.com/files/U91BM3EAE/F9LGR2SSY/nnb_redux_store_structure.js)